Toric/ConvexAnalysis/PolyhedralCone.lean

```diff
@@ -0,0 +1,42 @@
+/-
+Copyright (c) 2025 Paul Reichert. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Paul Reichert
+-/
+import Mathlib.Analysis.Convex.Cone.Pointed
+
+/-!
+# Pointed cone hull and polyhedral cones
+
+We define the pointed cone hull and what it means for a pointed cone to be polyhedral.
+-/
+
+variable {𝕜 E : Type*} [OrderedSemiring 𝕜] [AddCommMonoid E] [Module 𝕜 E]
+
+namespace PointedCone
+
+variable (𝕜) in
+/-- The span of a set `S` is the smallest pointed cone that contains `S`.
+Pointed cones being defined as submodules over nonnegative scalars, this is exactly the
+submodule span of `S` w.r.t. nonnegative scalars. -/
+abbrev span (S : Set E) : PointedCone 𝕜 E :=
+  Submodule.span _ S
+
+theorem subset_span {S : Set E} :
+    S ⊆ PointedCone.span 𝕜 S :=
+  Submodule.subset_span
+
+/-- A pointed cone is polyhedral if it is the convex hull of finitely many points. -/
+def IsPolyhedral (c : PointedCone 𝕜 E) : Prop :=
+  ∃ t : Finset E, c = PointedCone.span 𝕜 t
+
+protected def IsPolyhedral.span (s : Set E) (h : s.Finite) :
```

Comment:
Sorry, I thought I had commented on that already: Since the type of `h` mentions `s`, no need for `s` to be explicit:
```suggestion
protected def IsPolyhedral.span {s : Set E} (h : s.Finite) :
```

Reply:
Sorry, I missed that aspect of the suggested change where you proposed to replace `Fintype` with `Finite`.